tests/api/iotrace_stats_parser.py

```diff
@@ -0,0 +1,171 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from test_utils.size import Size, Unit, parse_unit
+from core.test_run import TestRun
+
+from iotrace import IotracePlugin
+
+
+class TraceStatistics:
+    def __init__(self, trace):
+        self.device = DeviceStats(trace["desc"]["device"])
+        self.duration = int(trace["duration"])
+        self.read = IOStatistics(trace["read"])
+        self.write = IOStatistics(trace["write"])
+        self.discard = ServiceStats(trace["discard"])
+        self.flush = ServiceStats(trace["flush"])
+        self.total = IOStatistics(trace["total"])
+
+    def __str__(self):
+        ret = f"{self.device}"
+        ret += f"duration: {self.duration}\n"
+        ret += f"duration: {self.read}"
+        ret += f"duration: {self.write}"
+        ret += f"duration: {self.discard}"
+        ret += f"duration: {self.flush}"
+        ret += f"duration: {self.total}"
+
+        return ret
+
+
+class DeviceStats:
+    def __init__(self, trace):
+        self.id = trace["id"]
+        self.name = trace["name"]
+        self.size = Size(float(trace["size"]), Unit.Blocks512)
+
+    def __str__(self):
+        ret = f"device: id: {self.id}, name: {self.name}, size {self.size}\n"
+        return ret
```

Comment:
You don't have to keep this string in variable. You could pass this directly to return.